modules/smtp/site.js

```diff
@@ -510,16 +487,16 @@ $(function () {
                 modalContentHeadline = "Your subject and body are empty!";
             }
 
-            // if contact_cc not exist in contact list for user
-            var checkInList = "";
-            if (list_emails) {
-                checkInList = check_cc_exist_in_contacts_list(e);
+            if (typeof check_cc_exist_in_contacts_list === 'function') {
```

Comment:
It's works, Just an idea about how this can be refactored, I think there is a way to check if the module is enabled, we can check like that .